integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/TransferDashFragment.kt

```diff
@@ -64,10 +65,9 @@ class TransferDashFragment : Fragment(R.layout.transfer_dash_fragment) {
     private val enterAmountToTransferViewModel by activityViewModels<EnterAmountToTransferViewModel>()
     private val transferDashViewModel by activityViewModels<TransferDashViewModel>()
     private val binding by viewBinding(TransferDashFragmentBinding::bind)
-    private var loadingDialog: FancyAlertDialog? = null
+    private var loadingDialog: AdaptiveDialog? = null
```

Comment:
`FancyAlertDialog` is being faded out. `AdaptiveDialog` has a progress version now too.